recipes/noaa-oisst/meta.yaml

```diff
@@ -0,0 +1,15 @@
+recipe:
+  python:
+    - file: recipe.py
+
+provenance:
+  provider: NOAA NCEI
+  website: https://www.ncdc.noaa.gov/oisst
+  reference: http://dx.doi.org/10.1175/1520-0442(1993)006%3C0768:IOMPAO%3E2.0.CO;2
+  license: ?
+
+extra:
+  maintainers:
```

Comment:
I basically just made all of this up. What process should we follow to design the meta.yaml file?

Reply:
Bakery section

```
bakery: aws-1
```

Reply:
Resources section

```
resources:
   memory: 10 GB
```

Reply:
Release tag, version, etc.

Reply:
We probably want a top-level `version` key indicating the version of the pangeo-forge metadata file itself, it case we want to evolve it over time.

Reply:
Also perhaps a `pangeo_forge` library version. If the API changes over time, it will be important to pin the recipe to a specific version of the library.